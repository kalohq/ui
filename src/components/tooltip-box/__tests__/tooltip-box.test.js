/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';

import TooltipBox from '../tooltip-box.react';

describe('TooltipBox', () => {
  test('should render properly', () => {
    const wrapper = renderer
      .create(<TooltipBox text="Hello! I am a tooltip">Hover me :)</TooltipBox>)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });

  test('should render properly - show right', () => {
    const wrapper = renderer
      .create(
        <TooltipBox text="Hello! I am a tooltip" show="right">
          Hover me :)
        </TooltipBox>
      )
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
