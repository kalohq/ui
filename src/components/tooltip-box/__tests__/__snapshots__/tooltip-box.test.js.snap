// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TooltipBox should render properly - show right 1`] = `
<div
  className="tooltip-container"
>
  Hover me :)
  <div
    className="tooltip-bubble tooltip-bubble--right"
  >
    Hello! I am a tooltip
  </div>
</div>
`;

exports[`TooltipBox should render properly 1`] = `
<div
  className="tooltip-container"
>
  Hover me :)
  <div
    className="tooltip-bubble"
  >
    Hello! I am a tooltip
  </div>
</div>
`;
